preview-src/index.adoc

```diff
@@ -1,5 +1,9 @@
 = Cloudflow, quickly develop, orchestrate, and operate distributed streaming applications on Kubernetes
+:description: Description goes here
```

Comment:
Should I complete the description that goes here?

Reply:
Yes, you would complete this in the adoc file for the home page. This is just the preview-src file that I work with when skinning.